### PR TITLE
Improve readability in multiply and primes tutorials

### PR DESCRIPTION
**Pull request by palmskog**

### CODE REVIEW

File: theories/Examples/Tutorial/PrimesComposition.v [hunk 1]
```diff
@@ -509,11 +511,11 @@ Qed.
 Lemma even_constrained_primes_composition_valid_message_char (m : Z) :
   valid_message_prop even_constrained_primes_composition m
     <->
-  (m > 1)%Z /\ Z.Even m \/ prime m.
+  ((m > 1 /\ Z.Even m) \/ prime m).
```
Comment (traiansf) [inline theories/Examples/Tutorial/PrimesComposition.v:514]: you can also drop the parentheses here

**Review by traiansf — APPROVED** [review]

File: theories/Examples/Tutorial/PrimesComposition.v (hunk 1 above)
Comment (palmskog) [reply theories/Examples/Tutorial/PrimesComposition.v:514]: @traiansf but don't these parentheses help people read the statement properly? In my view, it's often hard to parse any use of `\/` without parentheses. But if you insist, I can change to drop parentheses.

Comment (wkolowski) [reply theories/Examples/Tutorial/PrimesComposition.v:514]: I think the parentheses should stay.

**Review by wkolowski — APPROVED** [review]

File: theories/Examples/Tutorial/PrimesComposition.v (hunk 1 above)
Comment (traiansf) [reply theories/Examples/Tutorial/PrimesComposition.v:514]: Sorry, I misread. they are fine.
